src/AnalyzeAsm/Program.cs

```diff
@@ -1912,13 +1924,132 @@ static void CSEForAdrpAddAddresses(string inputFile, string outputFile)
             WriteResults(outputFile, resultBuilder.ToString());
         }
 
+        /// <summary>
+        /// Summarizes "ldr wzr
+        /// </summary>
+        static StringBuilder FindRedundantTest(string inputFile, ref int totalMethods, ref int foundMethods, ref int totalTestGroups)
+        {
+
+            /* JE/JNE checks for ZF
+             * Below instructions modify ZF
+             * add, and, dec, inc, lsl, neg, or, sal, sar, shl, shr, sbb, sub, xor
+             * 
+             * 
+             */
+            var testRegex = new Regex(@"test     (\w+), (\w+)");
+
+            //var aluRegex = new Regex(@"(add|and|dec|inc|lsl|neg|or|sal|sar|shl|shr|sbb|sub|xor)      (\w+),?");
+            //List<string> ops = new List<string>()
+            //{
+            //    " je ", " jne ", " jz ", "jnz "
+            //};
+
+            var aluRegex = new Regex(@"(add|btr|btc|bts|imul|mul|rol|ror|sal|sar|shl|shr|sbb|sub|xor)      (\w+),?");
+            List<string> ops = new List<string>()
+            {
+                " jb ", " jnb ", " jae ", "jnae "
```

Comment:
Why aren't `je`  et al. in this list?

Reply:
Or did you do two different passes...?

Reply:
I did it in 2 passes. With `jb`, etc. I didn't see any pattern. The patterns that I attached in dotnet/runtime#53053 are from `je`.